### PR TITLE
Sections 8.3-8.5 (RMA and AMO)

### PR DESCRIPTION
**Pull request by nspark**
This is the roll-up PR for the RMA and AMO Section Committee (Sections 8.3-8.5).

If you have changes to propose, please submit a PR against [nspark:section/rma-amo](https://github.com/nspark/specification/tree/section/rma-amo), open an issue, or add comments to this PR.

Commits for RCM-1: 1150f36...660e1c8

Commits for RCM-2: 660e1c8...03244f0
- Closes https://github.com/openshmem-org/specification/issues/147 (for this section)
- Closes https://github.com/openshmem-org/specification/issues/149 (for this section)

### CODE REVIEW

**Comment by nspark** [comment]
Here is the [RMA/AMO Section Committee draft](https://github.com/openshmem-org/specification/files/1265608/openshmem-1.4-draft-13b16bb.pdf) as of 13b16bb for the August 31 specification and mock release candidate meeting.

**Comment by nspark** [comment]
Here is the [RMA/AMO Section Committee draft](https://github.com/openshmem-org/specification/files/1283128/openshmem-1.4-draft-660e1c8.pdf) as of 660e1c8 for the RCM section draft deadline of Sept. 7.

We have one outstanding issue: https://github.com/openshmem-org/specification/issues/147.

RMA/AMO Section Committee members -- @BryantLam @min-si @davidozog -- please review and approve, as appropriate.

File: content/atomics_intro.tex [hunk 1]
```diff
@@ -1,44 +1,50 @@
...
+  \FUNC{shmem\_atomic\_fetch\_\{inc, add, and, or, xor\}}.
 
 \item
-The \textit{non-fetching} atomic routines update the remote memory in a single
-atomic operation.  A call to a non-fetching atomic routine issues the atomic
-operation and may return before the operation executes on the remote \ac{PE}.
-To force completion for these non-fetching atomic routines,
-\FUNC{shmem\_quiet}, \FUNC{shmem\_barrier}, or \FUNC{shmem\_barrier\_all} can
-be used by an \openshmem program.
+  The \textit{non-fetching} routines update the remote data object in a single
```
Comment (BryantLam) [inline content/atomics_intro.tex]: `\textit` --> `\emph` for fetching and non-fetching.

File: content/shmem_iput.tex [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 \begin{Fsynopsis}
 INTEGER dst, sst, nelems, pe
-CALL SHMEM_COMPLEX_IPUT(dest, source, dst, sst, nelems, pe)
+nCALL SHMEM_COMPLEX_IPUT(dest, source, dst, sst, nelems, pe)
```
Comment (nspark) [inline content/shmem_iput.tex]: Typo

Comment (jdinan) [inline content/shmem_iput.tex]: Typo "nCALL"

**Comment by nspark** [comment]
@BryantLam @min-si @davidozog: Please review my latest batch of commits (660e1c8...98b4d72) intended for RCM-2

**Comment by davidozog** [comment]
I approve of commits 660e1c8...98b4d72.

**Pull request merged**